### PR TITLE
Fix Typo and Correct Referent

### PR DESCRIPTION
**Pull request by RatanRSur**
Signed-off-by: Ratan Rai Sur <ratan.r.sur@gmail.com>

### CODE REVIEW

**Review by gem1no — APPROVED** [review]
looks good!

File: app/src/components/market/sections/market_create/steps/items/ask_question_step.tsx [hunk 1]
```diff
@@ -322,7 +322,7 @@ const AskQuestionStep = (props: Props) => {
                 disabled={!!loadedQuestionId}
                 name="question"
                 onChange={handleChange}
-                placeholder="What question do you want the world predict?"
+                placeholder="What question do you want the world to predict the answer of?"
```
Comment (Mi-Lan) [inline app/src/components/market/sections/market_create/steps/items/ask_question_step.tsx:325]: Either that or "What question do you want the world to predict?" @pimato Probably knows the best..

Comment (pimato) [reply app/src/components/market/sections/market_create/steps/items/ask_question_step.tsx:325]: I actually prefer: `What would you like to see the world predict?`

Comment (pimato) [reply app/src/components/market/sections/market_create/steps/items/ask_question_step.tsx:325]: Thank you @RatanRSur for pointing out the issue :)

**Review by pimato — APPROVED** [review]

**Pull request merged**